JetMETCorrections/Modules/test/JetResolutionDemo_cfg.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("myprocess")
+process.TFileService=cms.Service("TFileService", fileName=cms.string('JERplots.root'))
+
+##-------------------- Communicate with the DB -----------------------
+process.load('Configuration.StandardSequences.Services_cff')
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```

Comment:
`process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")`
I suggest this cfg to become an unit test.